### PR TITLE
Omitting root braces is no longer obsolete

### PR DESCRIPTION
**Pull request by trobro**

### CODE REVIEW

File: pages/syntax.md [hunk 1]
```diff
@@ -97,9 +97,11 @@ Quoted strings can be written with 'single' or "double" quotes.
 
 Note: single quotes are new (2017/7) and may not yet be supported on all platforms.
 
-#### Root Braces - **OBSOLETE**
+#### Root Braces
 
-While Hjson allows you to omit `{}` for the root object, this feature should no longer be used.
+Hjson allows you to omit `{}` for the root object, so that a config file contains nothing but keys and values, without indentation.
+
+Note: This feature was marked as deprecated for some years, but is now officially supported again. If you require maximum portability for your Hjson files, you are adviced to not omit root braces, in case third party implementations lack support for it.
```
Comment (dqsully) [inline pages/syntax.md:104]: How about this wording?
```suggestion
Hjson allows you to omit `{}` for the root object so that a config file can contain nothing but keys and values, similar to YAML.

Note: This feature was previously marked as deprecated but is now officially supported again. If you need maximum portability with your Hjson files, then we recommend you keep the root braces in case a third party implementation doesn't support it.
```

Comment (trobro) [reply pages/syntax.md:104]: Thanks for the suggestion, that sounds better, I'll commit it.
